src/bench/wallet_create_tx.cpp

```diff
@@ -115,6 +132,11 @@ static void WalletCreateTx(benchmark::Bench& bench, const OutputType output_type
     });
 }
 
-static void WalletCreateTxUseOnlyPresetInputs(benchmark::Bench& bench) { WalletCreateTx(bench, OutputType::BECH32); }
+static void WalletCreateTxUseOnlyPresetInputs(benchmark::Bench& bench) { WalletCreateTx(bench, OutputType::BECH32, /*allow_other_inputs=*/false,
+                                                                                        {{/*num_of_internal_inputs=*/4}}); }
+
+static void WalletCreateTxUsePresetInputsAndCoinSelection(benchmark::Bench& bench) { WalletCreateTx(bench, OutputType::BECH32, /*allow_other_inputs=*/true,
+                                                                                                    {{/*num_of_internal_inputs=*/4}}); }
 
-BENCHMARK(WalletCreateTxUseOnlyPresetInputs);
```

Comment:
In 08012f03bf2cc9ec4f8b21d5643f3423acb1b5d8 "bench: add benchmark for tx creation using pre-set inputs and coin selection"

nit: This should be in the previous commit.